test/token/ERC721/ERC721PausableToken.behavior.js

```diff
@@ -0,0 +1,90 @@
+const { assertRevert } = require('../../helpers/assertRevert');
+const { sendTransaction } = require('../../helpers/sendTransaction');
+
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+function shouldBehaveLikeERC721PausableToken ([owner, recipient, operator]) {
+  const firstTokenId = 1;
+  const data = '0x42';
+  const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+  const safeTransferFromWithData = function (token, from, to, tokenId, opts) {
```

Comment:
Do we actually need this helper? If not, I'd remove it entirely

Reply:
Do you mean to move helper's body to test (as it is not reused anywhere), or there is more simple way to call safeTransferFrom with extra arguments?